handbook/company/handbook.md

```diff
@@ -18,7 +18,7 @@ To contribute a new handbook page:
   a. [the "Company" handbook](https://fleetdm.com/handbook/company), or
   b. the handbook for a particular division (Engineering, Product Design, Customer Support, Sales, Marketing, Finance, Digital Experience)
 2. Locate the appropriate folder for the new page in [the GitHub repository under `handbook/`](https://github.com/fleetdm/fleet/tree/main/handbook).
-3. Create a new markdown file (like [one of these](https://github.com/fleetdm/fleet/tree/f90148abad96fccb6c5647a31877fa7e91b5ee57/handbook/digital-experience)).  A simple, easy way to do this is by clicking "Add file" on GitHub.com.
+3. Create a new markdown file (like [one of these](https://github.com/fleetdm/fleet/tree/f90148abad96fccb6c5647a31877fa7e91b5ee57/handbook/it-and-enablement)).  A simple, easy way to do this is by clicking "Add file" on GitHub.com.
```

Comment:
```suggestion
3. Create a new markdown file (like [one of these](https://github.com/fleetdm/fleet/tree/f90148abad96fccb6c5647a31877fa7e91b5ee57/handbook/digital-experience)).  A simple, easy way to do this is by clicking "Add file" on GitHub.com.
```

This updated link will not work correctly (the GitHub link is locked to a commit hash)